xaitk_saliency/utils/bin/sal_on_coco_dets.py

```diff
@@ -25,6 +25,11 @@
 @click.argument('coco_file', type=click.Path(exists=True))
 @click.argument('output_dir', type=click.Path(exists=False))
 @click.argument('config_file', type=click.File(mode='r'))
+@click.option(
+    '--fill',
+    help='RGB values (0-255) of fill to use when occluding three dimensional images, will fail if used with 1D image',
```

Comment:
```suggestion
    help='RGB values (0-255) of fill to use when occluding images, will fail if used with 1-channel images',
```

Is there an elegant way to support passing either RGB values or a grayscale value? I'm thinking longer-term of potential use cases where detection might need to be run on grayscale imagery (e.g. from IR sensors). I guess more broadly, is this tool designed specifically for RGB images, or it supposed to be more general?

Reply:
I chatted with @schencej earlier about using the click multiple value functionality which I think he is working on. With that you could provide something like `-f INT -f INT -f INT` instead of `--fill INT INT INT`, or alternatively just `-f INT` for a single-channel collection of images.